app/generator/content/part2.js

```diff
@@ -0,0 +1,120 @@
+const part2 = (statement) => {
+  const reductions = mapReductions(statement.funding)
+
+  return [
+    { text: 'Part 2. Calculation', style: 'header2' },
+    { text: `We calculated the total ${statement.scheme.shortName} payment amount by adding together payments for all the standards which are part of your agreement, as shown in this table.` },
```

Comment:
w/out causing you PTSD: is the assumption here tht 1 statement corresponds to 1 scheme?

Reply:
I wish someone could answer this.  So I've gone with the approach that this service will largely be consistent and independent of scheme.  For the current content it's pretty straight forward such as what you see here.

Most of part 2 is pretty generic too in terms of mapping.  Perhaps the area column would disappear display N/A for say Vets.

Part 3 will be a bit more wider change as it links to scheme guidance so maybe we'd have `part3SFI` `part3Vets` and just switch them.

However, later in the year I expect that a lot of this content will change anyway once we start getting real feedback.  That will likely happen long before we add new schemes.

So this repo will cover all schemes.  How complicated the mapping of content will be will depend.